ansible/roles/virtualbmc/tasks/domain.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Set common strings
+  set_fact:
+    # vbmcd should already be running, so --no-daemon stops vbmc from spawning
+    # another instance of the daemon.
+    cmd: "'{{ vbmc_virtualenv_path }}/bin/vbmc' --no-daemon"
+    log_arg: "--log-file '{{ vbmc_log_directory }}/vbmc-{{ domain }}.log'"
```

Comment:
Any reason not to append this to cmd here?

Reply:
Actually, no. I split it out in the first place because there was some weirdness with logging when the vbmc  commands themselves were forking a subprocess for the daemon, but now the daemon is managed by systemd it should all work in `cmd`.